Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -54,8 +55,13 @@ class PurchasesOrchestrator {
     private let identityManager: IdentityManager
     private let receiptParser: ReceiptParser
     private let deviceCache: DeviceCache
+    private let manageSubscriptionsModalHelper: ManageSubscriptionsModalHelper
+    private let beginRefundRequestHelper: BeginRefundRequestHelper
     private let lock = NSRecursiveLock()
 
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+    lazy var storeKit2Listener = StoreKit2TransactionListener(delegate: self)
```

Comment:
Same thread safe comment, I'm 99% sure this is fine, but wanted to double check.

Reply:
this one is annoying, but I couldn't a great workaround for it. I'm open to ideas. 
The problem is that you can't have a stored property that's only available on certain OS versions. 
Another approach would be to make the class available but no-op on older OS versions